python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_ishigami.py

```diff
@@ -95,8 +95,7 @@ def plotXvsY(sampleX, sampleY, figsize=(15, 3)):
 multivariateBasis = ot.OrthogonalProductPolynomialFactory(
     [im.X1, im.X2, im.X3])
 selectionAlgorithm = ot.LeastSquaresMetaModelSelectionFactory()
-projectionStrategy = ot.LeastSquaresStrategy(
-    inputTrain, outputTrain, selectionAlgorithm)
+projectionStrategy = ot.LeastSquaresStrategy(selectionAlgorithm)
```

Comment:
Unless I am mistaken, you are using an documented constructor here (and at other places):
![image](https://user-images.githubusercontent.com/31989332/178113544-80f1ac18-46fc-4f19-ab3c-4ae9ec69f24a.png)

You seem to be using `LeastSquaresStrategy(approxAlgoImpFact)`.

Reply:
yes, added this one to the list